airflow/providers/google/cloud/example_dags/example_bigquery_transfer.py

```diff
@@ -0,0 +1,82 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+Example Airflow DAG for Google BigQuery service.
+"""
+import os
+
+from airflow import models
+from airflow.providers.google.cloud.operators.bigquery import (
+    BigQueryCreateEmptyDatasetOperator, BigQueryCreateEmptyTableOperator, BigQueryDeleteDatasetOperator,
+)
+from airflow.providers.google.cloud.operators.bigquery_to_bigquery import BigQueryToBigQueryOperator
+from airflow.providers.google.cloud.operators.bigquery_to_gcs import BigQueryToGCSOperator
+from airflow.utils.dates import days_ago
+
+PROJECT_ID = os.environ.get("GCP_PROJECT_ID", "example-project")
+DATASET_NAME = os.environ.get("GCP_BIGQUERY_DATASET_NAME", "test_dataset_transfer")
+DATA_EXPORT_BUCKET_NAME = os.environ.get(
+    "GCP_BIGQUERY_EXPORT_BUCKET_NAME", "test-bigquery-sample-data"
+)
+ORIGIN = "origin"
+TARGET = "target"
+
+default_args = {"start_date": days_ago(1)}
+
+with models.DAG(
+    "example_bigquery_transfer",
+    default_args=default_args,
+    schedule_interval=None,  # Override to match your needs
+    tags=["example"],
+) as dag:
+    copy_selected_data = BigQueryToBigQueryOperator(
```

Comment:
Should we not move this to example_bigquery_to_bigquery.py/example_ bigquery_to_gcs? It would be nice if each module had a separate test. This will make it easier to manage it.

Reply:
I don't know. I would prefer to keep it here and have one DAG = one test. Splitting this into two dags will result in code duplication. It's not a big deal but still I prefer less code and fewer files.

Reply:
Done